miden-tx/src/testing/utils.rs

```diff
@@ -76,18 +70,3 @@ pub fn mock_executed_tx(asset_preservation: AssetPreservationStatus) -> Executed
         advice_witness,
     )
 }
-
-pub fn create_test_chain(created_notes: Vec<Note>) -> MockChain {
-    let mut mock_chain = MockChainBuilder::new().notes(created_notes).build();
-    mock_chain.seal_block();
-    mock_chain.seal_block();
-    mock_chain.seal_block();
-
-    mock_chain
-}
-
-pub fn build_dummy_tx_program() -> Program {
-    let operations = vec![Operation::Push(ZERO), Operation::Drop];
```

Comment:
replaced with the assembler parsing `begin push.0 drop end` which is way more understandable and better encapsulated.